ginivision/src/main/java/net/gini/android/vision/document/DocumentFactory.java

```diff
@@ -45,4 +45,9 @@ public static GiniVisionDocument newDocumentFromIntent(@NonNull final Intent int
     public static GiniVisionDocument newDocumentFromPhoto(@NonNull final Photo photo) {
         return ImageDocument.fromPhoto(photo);
     }
+
+    public static GiniVisionDocument newDocumentFromPhotoAndDocument(@NonNull final Photo photo,
```

Comment:
Had to add a new factory to be able to pass on non-Photo relevant data from an existing Document to the new one. In this case we need to pass on the Intent to know where the Document came from.